golem/gui/__init__.py

```diff
@@ -394,6 +396,217 @@ def screenshot_file2(project, test, execution, test_set, scr):
+                                    <td class="tests-ok">{total_cases_ok}</td>
+                                    <td class="tests-failed">{total_cases_fail}</td>
+                                    <td class="percentage">
+                                        <div class="progress">
+                                            <div aria-valuenow="10" style="width: 100%;" class="progress-bar pending pending-grey-background" data-transitiongoal="10"></div>
+                                            <div aria-valuenow="10" style="width: 0%;" class="progress-bar fail-bar failed-red-background" data-transitiongoal="10"></div>
+                                            <div aria-valuenow="20" style="width: 0%;" class="progress-bar ok-bar passed-green-background" data-transitiongoal="20"></div>
+                                        </div>
+                                    </td>
+                                    <td class="total-time">{time.strftime('%Mm %Ss', time.gmtime(total_time))}</td>
+                                    <td class="net-time">{time.strftime('%Mm %Ss', time.gmtime(net_elapsed_time))}</td>
+                                </tr>
+                            </tbody>
+                        </table>
+                    </div>
+                </div>
+            </div>
+            <h3 class="no-margin-top">Detail</h3>
+            <div class="widget widget-table">
+                        <div class="widget-content">
+                                <div class="table-responsive">
+                                        <table id="detailTable" class="table detail-table margin-bottom-5">
+                            <thead>
+                                <tr>
+                                    <th>#</th>
+                                    <th>Module</th>
+                                    <th>Test Name</th>
+                                    <th>Set Name</th>
+                                    <th>Environment</th>
+                                    <th>Browser</th>
+                                    <th>Result</th>
+                                    <th>Time</th>
+                                    <th>&nbsp;</th>
+                                </tr>
+                            </thead>
+                            <tbody>"""
+
+    count = 0
+    for test in execution_data['test_cases']:
+        test_name = test['name']
+        test_folder = test['test_set']
+        test_environment = test['environment']
+        test_full_name = test['full_name']
+        test_module = test['module']
+        test_browser = test['browser']
+        test_result = test['result']
+        test_set_name = test['set_name']
+        test_start_date_time = test['start_date_time']
+        test_sub_modules = test['sub_modules']
+        test_test_elapsed_time = test['test_elapsed_time']
+        test_case_data = report_parser.get_test_case_data(root_path, project, test_name,
```

Comment:
Here instead of test_name (test['name']) it should be test['full_name'] otherwhise the step section won't work when a test case is inside a subfolder, e.g.: test_name: 'some_test_name', full_name: 'folder.some_test_name'